packages/libs/pactjs-generator/src/contract/parsing/utils/dataWrapper.ts

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable @kadena-dev/no-eslint-disable */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+export interface IWrappedData<
+  T extends any = any,
+  N extends string | undefined = string | undefined,
+> {
+  name: N;
+  inspect: true;
+  data: T;
+}
```

Comment:
It could be nice to have a little more descriptive type argument names. E.g. `TData` and `TName`, what do you think?